### DISCUSSION TITLE
Add the option to show Image Carousels with a Slideshow template similar to the Media Carousel widget

### DISCUSSION THREAD

**Discussion by krann** [elementor/elementor#25442] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The 'Media Carousel' widget offers a 'Sideshow' setting, whereby a single 'hero' image is shown with a horizontal bar of scrollable thumbnails underneath. However, this does not lend itself to Advanced Custom Fields Pro. Conversely, the 'Gallery' widget WILL let you use ACF pro, but it WON'T offer a 'Slideshow' setting.

### Describe the solution you'd like.

I'd like a Slideshow setting in Gallery.

### Describe alternatives you've considered.

To get around the problem, I've used shortcode with the solution offered here:
https://sachinchoolur.github.io/lightslider/index.html
(there is a minor typo in the code, but I've fixed it and have it working here: https://www.ncestateagents.com/property/1-bedroom-third-floor-apartment-priced-to-sell-in-bogaz/. and other pages)

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
This request was also mentioned in the following issues and discussions:

- #9155
